src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -217,6 +219,46 @@ export default {
 				this.open = false
 			}
 		},
+		afterShow() {
+			const picker = this.$refs.picker
+			const input = picker.$refs.search.$el.querySelector('input')
+
+			// add focus trap in modal
+			picker.$el.addEventListener('keydown', this.checkKeyEvent)
+
+			if (input) {
+				input.focus()
+			}
+
+		},
+		checkKeyEvent(event) {
+			const picker = this.$refs.picker
```

Comment:
Shouldn't we escape early instead of wrapping all the code of this function in conditions?

```js
if (event.key !== 'Tab') {
    return
}
```

Reply:
Also, `event.preventDefault()` seems to be called in any cases. Might be better to have only one line for that.

Reply:
✅ i changed the process of the method. You think it works better now?

Reply:
Beside the extra console.log, it looks good :)

Reply:
Beside the extra console.log, it looks good :)

Reply:
:) removed